app/models/student.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Student < ApplicationRecord
+  validates_presence_of :first_name, :last_name, :cohort_id, :email, :alumni
+  validates :cohort_id, inclusion: { in: Cohort.list_ids, message: '%(value) is not a valid cohort' }
```

Comment:
**this needs to checked and tested once we have our tables are set up**. I think the list of `cohort_id` allowed could be a class method as implemented above and according to the documentation I read, I would return an error message (eg. "BE1964 is not a valid cohort")

Reply:
Okay, so line 5 is validating our foreign key id from the Cohort model, but we'd need to setup and test that model relationship to be sure, correct?

Reply:
yes, the relationship between the `cohort` and the `student` is tested with the `should belong_to :cohort` expectation